Traducir.Web.Net/Traducir.Web.Net.csproj

```diff
@@ -0,0 +1,61 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
can we use the new csproj format? so that we don't list each and every file ;)

Reply:
This is just the project as Visual Studio created it, of course we can tune it as appropriate. I'll need to investigate first what that new format it about, though. 😅